tutorials/snapshot-instances-jobs/index.mdx

```diff
@@ -1,238 +1,73 @@
+7. Set **cron schedule** to `0 2 * * *` and select the relevant time zone to run the job every day at 2:00 a.m. Refer to the [cron schedules documentation](/serverless-jobs/reference-content/cron-schedules/) for more information.
 
 8. Define the following environment variables:
-    - `INSTANCE_ID`: the ID of the Instance you want to snapshot
-    - `INSTANCE_ZONE`: the [Availabilitiy Zone](/instances/concepts/#availability-zone) of your Instance (e.g. `fr-par-2`)
-    - `SCW_ACCESS_KEY`: your API access key
-    - `SCW_SECRET_KEY`: your API secret key
-    - `SCW_DEFAULT_ORGANIZATION_ID`: your Organization ID
+    - `SCW_ACCESS_KEY`: your API access key.
+    - `SCW_SECRET_KEY`: your API secret key.
+    - `SCW_DEFAULT_PROJECT_ID`: your Project ID.
+    - `SCW_DEFAULT_ORGANIZATION_ID`: your Organization ID.
+    - `SCW_DEFAULT_REGION`: concerned region.
+
+For more details about variables used by `cli`, please refer to the [cli config documentation](https://github.com/scaleway/scaleway-cli/blob/master/docs/commands/config.md).
+
+9. In the **Execution** tab, define the desired command: `/scw block snapshot create volume-id=11111111-1111-1111-1111-111111111111` (replace the ID with your desired volume ID).
 
-9. Click **Create job**.
+10. Click **Create job**.
 
 ## Running the job
 
-From the **Overview** tab of the Serverless job you just created, click, **Actions**, then select **Run job** from the contextual menu.
+From the **Overview** tab of the Serverless job you just created, click **Actions**, then select **Run job** from the contextual menu.
 
 The execution appears in the **Job runs** section. You can access the logs of your job by clicking <Icon name="more" /> next to the job run ID, and selecting **See on Cockpit**.
 
 ## Possible improvements
 
 This tutorial is a lightweight example of how to create recurring snapshots of an Instance. You can go further by:
-- Using it to manage all your Instances snapshots
-- Create backups of your storage disks
-- Set up an alerting system in case of unexpected behavior
+- Using it to manage all your Instances' snapshots.
+- Creating backups of your storage disks.
+- Setting up an alerting system in case of unexpected behavior.
 
 ## Additional resources
 
-- [Serverless Jobs Documentation](/serverless-jobs/how-to/create-job/)
-- [Other methods to deploy Jobs](/serverless-jobs/reference-content/deploy-job/)
-- [API keys documentation](/iam/how-to/create-api-keys/)
-- [CRON schedule reference](/serverless-jobs/reference-content/cron-schedules/)
+- [Serverless Jobs Documentation](/serverless-jobs/how-to/create-job/).
+- [Other methods to deploy Jobs](/serverless-jobs/reference-content/deploy-job/).
+- [API keys documentation](/iam/how-to/create-api-keys/).
+- [CRON schedule reference](/serverless-jobs/reference-content/cron-schedules/).
```

Comment:
```suggestion
- [CRON schedule reference](/serverless-jobs/reference-content/cron-schedules/)
```

Reply:
These bullet points are not full sentences, so should not have periods at the end.
The other example above is debatable.